debian/ubuntu-advantage-tools.postinst

```diff
@@ -4,14 +4,6 @@ set -e
 
```

Comment:
I think all trusty things in postinst need to stay, because until trusty is well past end of ESM, we could still have people upgrading from trusty to later releases

Reply:
I kept there the data which relates to upgrades - things as `ESM_INFRA_OLD_APT_SOURCE_FILE_TRUSTY` - so the `postinst` can change stuff when you leave Trusty.
Those two checks I removed expect that you are installing on Trusty (e.g. the current `VERSION_ID` is `14.04`). When updating, you will be in a different version anyway

Reply:
Ah excellent point, I'll take a closer look